jcl/src/java.base/share/classes/jdk/internal/nicl/NativeInvoker.java

```diff
@@ -175,5 +183,73 @@ private static long getStructAddress(Reference<?> arg) throws IllegalAccessExcep
 		Pointer<?> ptr = RuntimeSupport.createPtr(addr, returnLayoutType);
 		return (Reference<?>) ptr.deref();
 	}
+	
+	private static long generateFunctionPointerBinding(Object anonInstance) {
```

Comment:
Why `anonInstance`?  Should this be `instance` or some other similar name?  The `anon` part implies that this is from `Unsafe.defineAnonymousClass` when its just as valid for a user to handwrite a functional interface